RadixWallet/Clients/EntitiesVisibilityClient/EntitiesVisibilityClient+Interface.swift

```diff
@@ -1,46 +1,32 @@
 // MARK: - EntitiesVisibilityClient
 /// Controls the visibility of the entities in the Wallet
 public struct EntitiesVisibilityClient: Sendable {
-	public var hideAccounts: HideAccounts
-	public var hidePersonas: HidePersonas
-	public var unhideAllEntities: UnhideAllEntities
-	public var getHiddenEntityCounts: GetHiddenEntityCounts
+	public var hideAccount: HideAccount
+	public var hidePersona: HidePersona
+	public var unhideAccount: UnhideAccount
+	public var unhidePersona: UnhidePersona
+	public var getHiddenEntities: GetHiddenEntities
 }
 
 extension EntitiesVisibilityClient {
-	public struct HiddenEntityCounts: Hashable, Sendable {
-		public let hiddenAccountsCount: Int
-		public let hiddenPersonasCount: Int
+	public struct HiddenEntities: Hashable, Sendable {
+		public let accounts: Accounts
+		public let personas: Personas
 	}
 
-	public typealias HideAccounts = @Sendable (Set<Account.ID>) async throws -> Void
-	public typealias HidePersonas = @Sendable (Set<Persona.ID>) async throws -> Void
-	public typealias UnhideAllEntities = @Sendable () async throws -> Void
-	public typealias GetHiddenEntityCounts = @Sendable () async throws -> HiddenEntityCounts
+	public typealias HideAccount = @Sendable (Account.ID) async throws -> Void
+	public typealias HidePersona = @Sendable (Persona.ID) async throws -> Void
+	public typealias UnhideAccount = @Sendable (Account.ID) async throws -> Void
+	public typealias UnhidePersona = @Sendable (Persona.ID) async throws -> Void
+	public typealias GetHiddenEntities = @Sendable () async throws -> HiddenEntities
 }
 
 extension EntitiesVisibilityClient {
-	public func hideAccounts(ids: some Collection<Account.ID>) async throws {
-		try await hideAccounts(Set(ids))
+	func hide(account: Account) async throws {
+		try await hideAccount(account.id)
```

Comment:
Why do we need both `hide(account: Account)` and `hideAccount`? Isn't the latter enough? Come to that, isn't it enough to have `hideEntity`?

Reply:
`hideEntity` doesn't work since one uses `Account.ID` and the other `Persona.ID`. The second method is just an inherited helper method used to call from the Account object instead than from its id. Can definitely be removed.

Reply:
Hm, I thought we still had an Entity address that could be cast to either of those two, perhaps not.